foxpuppet/tests/test_browser_model.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from foxpuppet import FoxPuppet
+
+
+# Tests opening a new private browsing window via menu
```

Comment:
With stronger rules set for pep8 the docstring might even have to look like "Test opening a new private browsing window via menu.".